labman/db/process.py

```diff
@@ -685,6 +767,9 @@ def create(cls, user, plates, plate_ext_id, robot):
             raise ValueError(
                 'Cannot compress %s gDNA plates. Please provide 1 to 4 '
                 'gDNA plates' % len(plates))
+
+        volume = 1  # TODO: where did this magic # come from and can it change?
```

Comment:
Do we have an issue to figure out if `volume = 1` is correct?

Reply:
Didn't before, but now we do :)  ... just added https://github.com/jdereus/labman/issues/341